requirements.txt

```diff
@@ -0,0 +1 @@
+awscli==1.7.25
```

Comment:
@kbaum, It installs `awscli` when you push you code to heroku if you are in python application. 
for ruby we can include a gem file with with this content `gem 'aws-cli', '~> 0.0.1.pre4'` . Providing both the `requirement.txt` and the `gemfile` will be ok.